MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -34,50 +34,156 @@
 
 @interface MSALPublicClientApplication : NSObject
 
+/*! Whether or not to validate the authority used with the application, default YES */
```

Comment:
Might want to expand this. To a developer fresh from the field, "authority" concept will be unclear, as is unclear what is meant by "validate."